lib/themes/dosomething/paraneue_dosomething/includes/preprocess/preprocess_page.inc

```diff
@@ -234,17 +234,6 @@ function paraneue_dosomething_preprocess_page(&$vars) {
     'terms' => dosomething_user_get_link_terms_service(),
   );
 
-  // Footer affiliate logo.
-  $footer_logo_enabled = theme_get_setting('footer_affiliate_logo');
```

Comment:
yaaaa, we use it on those sites, but they are many many versions behind in the code base. 
 ![](https://i.imgur.com/c6QQqwx.png)

I think we should maybe leave this in, since/if we fold those instances into our own.